contracts/cw-ics721-bridge/src/msg.rs

```diff
@@ -135,6 +135,40 @@ pub enum QueryMsg {
     /// Gets this contract's cw721-proxy if one is set.
     #[returns(Option<::cosmwasm_std::Addr>)]
     Proxy {},
+
+    /// Gets the code used for instantiating new cw721s.
+    #[returns(u64)]
+    Cw721CodeId {},
+
+    /// Gets a list of classID (from NonFungibleTokenPacketData) and
+    /// cw721 contract we have instantiated for that classID.
+    #[returns(Vec<ClassIdToNftContractResponse>)]
+    ClassIdToNftContract {},
```

Comment:
this query already exists. see the above `NftContract` query. if you prefer this naming, feel free to rename it.

Reply:
This query returns a list of all class ids and their collection address counterpart. `NftContract` returns a specific collection address for a given class id.

Reply:
Pagination has been added. Pls check changes in query_class_id_to_nft_contract().